### DISCUSSION TITLE
Ability to filter stacks by agent and/or Status

### DISCUSSION THREAD

**Discussion by danielpetrica** [louislam/dockge#450] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

I only found this similar feature request https://github.com/louislam/dockge/discussions/114 but as it implements edits on the data of stacks I think they may be seen as different features that may connect

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

It would be awesome to be able to filter the list of stacks based on:

- Agents
- Stacks
- Tags eventually if this feature request is implemented https://github.com/louislam/dockge/discussions/114

### ✔️ Solution

Implement an add filters button on the left of the search row. The button should open a modal allowing you to filter the results below. I think it can be implemented all in the front-end.

**Comment by danielpetrica** [comment]
If it can be implemented/is in line with project idea I can implement it

**Comment by moenster** [comment]
I would love this feature to be added. In my small setup I only have three remote agents and 27 stacks in total, and it's a bit hard to get a good overview in the left hand pane.
